spec/controllers/orders_controller_spec.rb

```diff
@@ -0,0 +1,100 @@
+require "rails_helper"
+
+RSpec.describe OrdersController, type: :controller do
+  describe "GET #new" do
+    subject { get :new, event_id: event.id }
+
+    context "this event has expired" do
+      let!(:event) { create(:event, starts_at: 1.week.ago) }
+
+      it "is successful" do
+        subject
+        expect(response).to be_success
+        expect(assigns(:event)).to eq event
+      end
+    end
+
+    context "this event has not expired" do
+      let!(:event) { create(:event, starts_at: 1.week.since) }
```

Comment:
Let's create unexpired events by default, so this should be `create(:event)` only